code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -404,6 +404,9 @@ GLOBAL_LIST_EMPTY(lifts)
 				var/datum/callback/land_slam = new(collided, /mob/living/.proc/tram_slam_land)
 				collided.throw_at(throw_target, 200 * collision_lethality, 4 * collision_lethality, callback = land_slam)
 
+				//Add to the blood counter. Notify the signs that the tram hit someone.
```

Comment:
This comment could rot, and also is just a bit redundant

Reply:
Removed.